WORKSPACE

```diff
@@ -1 +1,5 @@
-# Empty WORKSPACE file.
+workspace(name = "rules_java")
+
+load("//java:repositories.bzl", "rules_java_dependencies", "rules_java_toolchains")
```

Comment:
Can we use `@rules_java//java:repositories.bzl` here so this part can be easily copy-pasted?

Reply:
Sure, done!